.eslintrc

```diff
@@ -7,9 +7,16 @@
       "error",
       "ForInStatement",
       "LabeledStatement",
-      "WithStatement",
+      "WithStatement"
     ],
-    "operator-linebreak": ["error", "after"]
+    "operator-linebreak": ["error", "after"],
```

Comment:
I needed to copy the default rule configuration from air-bnb to disable only `ignoreComments` option.

Reply:
@esakal 👍